packages/flutter/lib/src/widgets/editable.dart

```diff
@@ -341,6 +374,13 @@ class RawInputLineState extends ScrollableState<RawInputLine> {
     else if (_cursorTimer != null && (!focused || !config.value.selection.isCollapsed))
       _stopCursorTimer();
 
+    if (_selectionHandles != null && !focused) {
+      Timer.run(() { // can't hide while building, since it triggers a rebuild
```

Comment:
scheduleMicrotask again for the same reason.  For example, you wouldn't want user input to sneak in during the async gap because the selection handles would probably get mad if the user interacted with them in that state.